src/index.ts

```diff
@@ -18,6 +18,7 @@ import ImageTiles from './blocks/ImageTiles/ImageTiles'
 import ImageCaption from './components/ImageCaption/ImageCaption'
 import Layout from './layouts/Layout/Layout'
 import LinkWrapper from './components/LinkWrapper/LinkWrapper'
+import Login from './blocks/Login/Login'
```

Comment:
Good job on remember to add it to the global exports! I forget to do this all the time heheh